core/Haris.HostApp/app.config

```diff
@@ -1,5 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <connectionStrings>
+    <add name="db" connectionString="mongodb://localhost:27017/CubeServer"/>
```

Comment:
Myślałem, żeby dodać biny mongo do solucji i spróbować to postawić jako moduł. Co o tym myślisz ?